tests/yast2_cmd/yast_tftp_server.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: this test checks that YaST's tftp-server module are
+#          setup, enabled and disabled correctly in cmd mode.
+# Maintainer: Shukui Liu <skliu@suse.com>
+
+package yast_tftp_server;
```

Comment:
`package yast_tftp_server;` is not needed, you will not use anything from here anywhere else

Reply:
`package yast_tftp_server;` is deleted.